content/resources/how-to-guides/how-to-zapier-alert.md

```diff
@@ -0,0 +1,117 @@
+---
+title: How to Alert Task to Zapier Webhook
+description: >
+  Learn how to use webhooks with InfluxDB to send alerts to Zapier
+menu:
+  resources:
+    parent: How-to guides
+weight: 102
+---
+### How To: Alert Task to Zapier Webhook
+
+### By: Tyler Nguyen
+
+If you have been a long time user of InfluxDB, the role of tasks within the platform is, without a doubt, a
+
+spotlight feature of the product. Tasks allow users to automatically execute a bit of Flux code: 
```

Comment:
Don't double-space markdown. Each line renders as a separate paragraph. Markdown paragraphs need to be single-spaced.